sky/clouds/service_catalog/azure_catalog.py

```diff
@@ -68,6 +68,10 @@ def get_region_zones_for_instance_type(instance_type: str,
 
 
 def get_gen_version_from_instance_type(instance_type: str) -> Optional[int]:
+    if 'Generation' in _df.columns:
+        return _df[_df['InstanceType'] == instance_type]['Generation'].iloc[0]
+
+    # Backward compatibility for the older catalog.
```

Comment:
I think a few months ago we had a discussion about how to handle catalog schema changes. And I think we eventually agreed to @romilbhardwaj's opinion that adding `if` statements to maintain backward compatibility is not a good idea. So why don't we bump up the catalog version to v3 and remove the code for backward compatibility?

Reply:
Sounds good. Just increased the version number. PTAL